apps/admin/backend/schema.sql

```diff
@@ -42,13 +42,23 @@ create table cvrs (
   ballot_type text not null 
     check (ballot_type = 'absentee' or ballot_type = 'precinct' or ballot_type = 'provisional'),
   batch_id text not null,
-  batch_label text not null,
   party_id text,
   precinct_id text not null,
   scanner_id text not null,
   sheet_number integer check (sheet_number is null or sheet_number > 0),
   votes text not null,
   created_at timestamp not null default current_timestamp,
+  foreign key (election_id) references elections(id)
+    on delete cascade,
+  foreign key (batch_id, scanner_id) references scanner_batches(batch_id, scanner_id)
+);
+
+create table scanner_batches (
+  batch_id text not null,
+  label text not null,
+  scanner_id text not null,
```

Comment:
If all CVRs in a batch have the same scannerId, then maybe we can remove it from the `cvrs` table and join `scanner_batches` whenever we need it?

Reply:
(And maybe the same applies to `precinct_id` if that makes sense to move into `scanner_batches`?)

Reply:
Good idea on the `scanner_id`. On the `precinct_id`, I think that breaks down with the "All Precincts" use case. We have CVRs from multiple precincts in the same batch and need to keep track of each's precinct individually.

Reply:
Ah, that makes sense, re: `precinct_id` - thanks for the context!